react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/views/ComposeViewWrapper.kt

```diff
@@ -64,4 +100,75 @@ abstract class ComposeViewWrapper<T : View> : FrameLayout {
             wrappedView?.requestLayout()
         }
     }
+
+    private fun setComposeOwnersOnFrameLayoutContainerIfNeeded() {
+        var parent = this.parent
+        var level = 0
+
+        while (parent != null && level < 10) { // Limit search depth
+            if (parent is FrameLayout) {
+                setComposeOwnersOnView(parent)
```

Comment:
If we set it on the parent, do we also need to set it on the parent of that parent? And if so, do we need to reach the root? Not sure if a level of 10 may be arbitrary in that case...

Reply:
It is completely arbitrary to not go too far. It's a good point though. Because in my tests it ended up being set in a couple parents, but I didn't test with just setting it to the closest parent. I will see if that works